R/ieee_article.R

```diff
@@ -3,8 +3,9 @@
 #' Format for creating submissions to IEEE Transaction journals. Adapted from
 #' \url{http://www.ieee.org/publications_standards/publications/authors/author_templates.html}.
 #'
-#' Presently, only the \code{"conference"} paper mode offered by the
-#' \code{IEEEtran.cls} is supported.
+#' Presently, only the \code{"conference"} paper mode offered by
+#' \code{IEEEtran.cls} is fully supported, with experimental support for the
+#' \code{"journal"} mode.
```

Comment:
We are saying that the mode for journal is experimental, but the skeleton use Journal mode by default. 

We may need to make it clear what needs to be kept or not. 

We could also revert back to a skeleton for conference mode first, then show what to add for journal mode. I don't want IEEE template user to be lost. 

Is journal or conference the most used from your experience ?